src/chess_board.rs

```diff
@@ -180,50 +184,60 @@ impl ChessBoard
                 let piece = BitboardPiece::from_file_rank(x, 8 - y - 1);
                 if self.allies.contains(piece)
                 {
-                    print!("\x1b[1;32m");
+                    print!("{}", color::gen_color_escape_code(color::FG_GREEN, is_white_space));
```

Comment:
Need to decouple color functionality and board state. So remove the bool parameter to `color::gen_color_escape_code()` and add a background color as a second parameter.